lib/games_collection.rb

```diff
@@ -29,4 +30,14 @@ def biggest_blowout
     blowout = @games.max_by { |game| (game.home_goals - game.away_goals).abs }
     (blowout.home_goals - blowout.away_goals).abs
   end
+
+  def percentage_home_wins
+    home_wins = @games.find_all { |game| game.home_goals > game.away_goals}
+    (home_wins.count.to_f / @games.count.to_f).round(2)
+  end
+
+  def percentage_visitor_wins
```

Comment:
Think about refactor op to change count to .length. count is an enum, length is not.  Higher performance with length.